test/integration/node-specific/abort_signal.test.ts

```diff
@@ -70,17 +73,37 @@ describe('AbortSignal support', () => {
     await client.connect();
     db = client.db('abortSignal');
     collection = db.collection('support');
+
+    client.on('commandSucceeded', ev => {
+      if (
+        ev.commandName === 'find' ||
+        ev.commandName === 'aggregate' ||
+        ev.commandName === 'listCollections'
+      ) {
+        const cursorId = Long.isLong(ev.reply.cursor.id)
```

Comment:
I thought that, with OnDemandDocument, cursorIds were always Longs now.  Is there a case where that isn't true?

Reply:
This is from command monitoring which, to prevent a backwards breaking change, deserializes the whole response with the user's BSON settings.